cmd/kubeadm/app/master/manifests.go

```diff
@@ -367,6 +362,20 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	return command
 }
 
+func getEtcdCommand(cfg *kubeadmapi.MasterConfiguration) []string {
+	var command []string
+
+	defaultArguments := map[string]string{
+		"listen-client-urls":    "http://127.0.0.1:2379",
+		"advertise-client-urls": "http://127.0.0.1:2379",
+	}
+
+	command = append(command, "etcd", "--data-dir=/var/lib/etcd")
```

Comment:
I would be ok with treating the data dir special; after all that's the heart of our cluster.
So for me you could add `DataDir string` to the struct and use it everywhere (do a search for /var/lib/etcd and use that val there instead).

You can make `/var/lib/etcd` a constant and use it as the default across the board.